tests/e2e/config.ts

```diff
@@ -0,0 +1,14 @@
+const config = {
+  // environment
+  baseUrlOcis: process.env.BASE_URL_OCIS ?? 'https://localhost:9200',
```

Comment:
would sound better if we write `baseOcisUrl`

Reply:
What concerns English language, I guess ocisBaseUrl would be the nicest way of calling this variable. However since the env variable name is BASE_URL_OCIS, I think it's less confusing to keep it as it is.